<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_table">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>361</width>
      <height>511</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>DAC0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DAC15</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Vset</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vmon</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Imon</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>31</string>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string>32</string>
     </property>
    </item>
    <item row="10" column="2">
     <property name="text">
      <string>33</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>34</string>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string>35</string>
     </property>
    </item>
    <item row="11" column="2">
     <property name="text">
      <string>36</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>37</string>
     </property>
    </item>
    <item row="12" column="1">
     <property name="text">
      <string>38</string>
     </property>
    </item>
    <item row="12" column="2">
     <property name="text">
      <string>39</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item row="13" column="1">
     <property name="text">
      <string>41</string>
     </property>
    </item>
    <item row="13" column="2">
     <property name="text">
      <string>42</string>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string>43</string>
     </property>
    </item>
    <item row="14" column="1">
     <property name="text">
      <string>44</string>
     </property>
    </item>
    <item row="14" column="2">
     <property name="text">
      <string>45</string>
     </property>
    </item>
    <item row="15" column="0">
     <property name="text">
      <string>46</string>
     </property>
    </item>
    <item row="15" column="1">
     <property name="text">
      <string>47</string>
     </property>
    </item>
    <item row="15" column="2">
     <property name="text">
      <string>48</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_initialVoltage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>1.75</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial Voltage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_runProgramm">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Programm</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_logView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>651</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_logView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Log View</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_logView">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
